src/settings.ts

```diff
@@ -14,7 +14,12 @@ export const PLUGIN_NAME = "homebridge-ratgdo";
 export const RATGDO_AUTODISCOVERY_INTERVAL = 60;
 
 // mDNS TXT record project name associated with a Ratgdo device.
-export const RATGDO_AUTODISCOVERY_PROJECT_NAMES: string[] = [ RatgdoVariant.KONNECTED, RatgdoVariant.RATGDO ];
+export const RATGDO_AUTODISCOVERY_PROJECT_NAMES: string[] = [
+  `${RatgdoVariant.KONNECTED}-gdov1-s`,
```

Comment:
Two things here:

* Please avoid the use of template literals...I have a strong aversion to them. 😄
* Please use as much of a line as possible...I prefer not to waste horizontal space.

Thanks!

Reply:
> Please use as much of a line as possible...I prefer not to waste horizontal space.
I made changes to this line, however I'm unsure if it fits your definition of wasting horizontal space. Please let me know if you want me to change it to:
```
export const RATGDO_AUTODISCOVERY_PROJECT_NAMES: string[] = [ RatgdoVariant.KONNECTED + "-gdov1-s", RatgdoVariant.KONNECTED + "-gdov2-q",
  RatgdoVariant.KONNECTED + "-gdov2-s", RatgdoVariant.RATGDO ];
```